clients/include/utility.hpp

```diff
@@ -131,26 +97,21 @@ inline void rocblas_print_matrix(
     for(size_t i = 0; i < m; i++)
         for(size_t j = 0; j < n; j++)
         {
-            printf("matrix  col %d, row %d, CPU result=%f, GPU result=%f\n",
-                   i,
-                   j,
-                   CPU_result[j + i * lda],
-                   GPU_result[j + i * lda]);
+            std::cout << "matrix  col " << i << ", row " << j
+                      << ", CPU result=" << CPU_result[j + i * lda]
```

Comment:
should this be rocblas_cout in place of std::cout ?

Reply:
Ignore all rocblas_cout/cerr and std::cout/cerr changes as they are mechanical.